### PR TITLE
Addd removeVolumes option for Docker Template config.

### PR DESCRIPTION
**Pull request by KostyaSha**
Closes #253

### CODE REVIEW

File: docker-plugin/src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/config.jelly [hunk 1]
```diff
@@ -77,6 +77,8 @@
         <f:textbox/>
     </f:entry>
 
-
+    <f:entry title="Remove volumes" field="removeVolumes">
```
Comment (lanwen) [inline docker-plugin/src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/config.jelly]: `"${%Remove volumes after run}"` ?

Comment (KostyaSha) [inline docker-plugin/src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/config.jelly]: Not critical also i want migrate to groovy later.

Comment (KostyaSha) [inline docker-plugin/src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/config.jelly]: fixed

**Comment by lanwen** [comment]
:+1:

**Pull request merged**